types/wlr_output_damage.c

```diff
@@ -58,6 +58,7 @@ struct wlr_output_damage *wlr_output_damage_create(struct wlr_output *output) {
 	}
 
 	output_damage->output = output;
+	output_damage->max_rects = 20;
```

Comment:
I'd avoid magic constants in the code:
```
output_damage->max_rects = WLR_OUTPUT_DAMAGE_MAX_RECTS;
```
and then define it in a wlr_output.h.

Reply:
Well, it's not a constant, it's a default value. Does it make sense to add something like `WLR_OUTPUT_DAMAGE_DEFAULT_MAX_RECTS`?

Reply:
Sure it's a default but that's a compile time constant. I'd do the #define nevertheless since it's more descriptive and can be reused at other locations if needed.